File: src/main/java/com/example/usercrud/presentation/UIController.java
package com.example.usercrud.presentation;

import lombok.extern.slf4j.Slf4j;
import org.springframework.stereotype.Controller;
import org.springframework.ui.Model;
import org.springframework.web.bind.annotation.GetMapping;


@Controller
@Slf4j
public class UIController {
    @GetMapping("/createuser")
    public String addUser() {
        return "create-user";
    }

    @GetMapping("/updateuser")
    public String updateUser() {
        return "update-user";
    }
}

File: src/main/kotlin/cloud/folium/usercrud/presentation/UIController.kt
package cloud.folium.usercrud.presentation

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.stereotype.Controller

@Controller
class UIController {
    @GetMapping("/createuser")
    fun addUser(): String {
        return "create-user"
    }

    @GetMapping("/updateuser")
    fun updateUser(): String {
        return "update-user"
    }
}